Recrutify/Recrutify.Services/Services/FormEmailService.cs

```diff
@@ -60,25 +62,52 @@ public IEnumerable<EmailRequest> GetEmailRequestsForSendTest(IEnumerable<Candida
             return emailRequests;
         }
 
-        public IEnumerable<EmailRequest> GetEmailRequestsForInterviewInvite(IEnumerable<CandidateDTO> candidates, DateTime interviewTime, string templatePath, string interviewType)
+        public IEnumerable<EmailRequest> GetEmailRequestsForInterviewInvite(IEnumerable<Interview> interviews)
         {
-            var emailRequests = new List<EmailRequest>();
-            var generator = CreateGenerator(templatePath);
-            foreach (var candidate in candidates)
-            {
-                var emailMessage = new EmailRequest();
-                emailMessage.Subject = "Interview";
-                emailMessage.Body = generator.Render(new
-                {
-                    name = candidate.Name,
-                    dateTime = interviewTime.ToString("MM/dd/yyyy HH:mm tt"),
-                    interviewerType = interviewType,
-                });
-                emailMessage.ToEmail = candidate.Email;
-                emailRequests.Add(emailMessage);
-            }
+            var mailMessages = new List<EmailRequest>();
+            var generatorForCandidate = CreateGenerator(Constants.TemplatePath.InterviewTemplate);
+            var generatorForUser = CreateGenerator(Constants.TemplatePath.InterviewerTemplate);
 
-            return emailRequests;
+            return interviews.Select(x => CreateEmailRequest(
+                                                x.Candidate.Email,
+                                                CreateInviteFile(x.AppointDateTimeUtc, "Exadel: Invitation for an interview"),
+                                                generatorForCandidate
+                                                .Render(
+                                                    new
+                                                    {
+                                                        name = x.Candidate.Name,
+                                                        interviewerType = x.InterviewType.GetDescription(),
+                                                        dateTime = x.AppointDateTimeUtc.AddHours(3).ToString(),
+                                                    })))
+                .Union(
+                interviews.Select(x => CreateEmailRequest(
+                                                x.User.Email,
+                                                CreateInviteFile(
+                                                    x.AppointDateTimeUtc,
+                                                    $"Candidate: {x.Candidate.Name}" +
```

Comment:
Put forming description in separate method

Reply:
+